nemo_text_processing/text_normalization/hi/data/numbers/thousands.tsv

```diff
@@ -0,0 +1,8 @@
+१०००	हज़ार
```

Comment:
I'm confused what this data file is for. The trailing 0s are supposed to be managed by your cardinal class. This should just have matching between the thousands place digit.